### PR TITLE
Intl.NumberFormat v3's default roundingMode should be "halfExpand"

### PR DESCRIPTION
**Pull request by Constellation**
The test was using "halfExpand (default)".

### CODE REVIEW

**Comment by Constellation** [comment]
PTAL @rwaldron  :)

**Comment by rwaldron** [comment]
@sffc can you do a quick review here?

File: test/intl402/NumberFormat/prototype/resolvedOptions/roundingMode.js [hunk 1]
```diff
@@ -9,7 +10,7 @@ features: [Intl.NumberFormat-v3]
 var options;
 
 options = new Intl.NumberFormat([], {}).resolvedOptions();
-assert.sameValue(options.roundingMode, 'halfExpand (default)');
+assert.sameValue(options.roundingMode, 'halfExpand');
```
Comment (jugglinmike) [inline test/intl402/NumberFormat/prototype/resolvedOptions/roundingMode.js:13]: This should use the third argument of `sameValue`, e.g.

```js
assert.sameValue(options.roundingMode, 'halfExpand', 'default');
```

Doing so will disambiguate failures from a subsequent assertion.

**Review by sffc — APPROVED** [review]

File: test/intl402/NumberFormat/prototype/resolvedOptions/roundingMode.js (hunk 1 above)
Comment (Constellation) [reply test/intl402/NumberFormat/prototype/resolvedOptions/roundingMode.js:13]: Sounds good! Changed.

**Comment by Constellation** [comment]
@rwaldron OK, it is reviewed, can you merge it?

**Review by rwaldron — APPROVED** [review]

**Pull request merged**